### PR TITLE
Borrow Form and Receipt draft

### PR DESCRIPTION
**Pull request by edjinjerney**
# Borrow Form and Receipt draft

## Details

Created initial front-end design for the borrow request form and request receipt

## Et al. info

- Does not exactly follow the mockup designs yet
- Not fully responsive

### CODE REVIEW

**Comment by edjinjerney** [comment]
hello

File: src/app/(general)/borrowform/borrowform.module.scss [hunk 1]
```diff
@@ -0,0 +1,231 @@
+@import "/Users/Tuff Gaming/Documents/Programming Files/Nex.Tech/Nex.Tech/src/styles/variables.scss";
```
Comment (1123Alessandro) [inline src/app/(general)/borrowform/borrowform.module.scss]: convert to relative directory

**Review by 1123Alessandro — APPROVED** [review]

**Review by 1123Alessandro — CHANGES_REQUESTED** [review]
labels for the PR

**Review by 1123Alessandro — APPROVED** [review]
lgtm

File: prisma/seed.js
Comment (carlmitzchel) [inline prisma/seed.js:1]: Might cause conflicts when merging to main

**Review by carlmitzchel — APPROVED** [review]
Make sure to change .env and seed.js codes when using main branch

**Pull request merged**